terraform/eks-cloudwatch/variables.tf

```diff
@@ -0,0 +1,22 @@
+# ------------------------------------------------------------------------
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+# -------------------------------------------------------------------------
+
+variable "eks_cluster_name" {
+  default = "aws-otel-testing-framework-eks"
+}
+
+variable "mock_endpoint" {
+  default = "localhost/put-data"
```

Comment:
would it be a common var in the common.tf?

Reply:
Code cleansing done